flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/HighAvailabilityServices.java

```diff
@@ -212,19 +212,17 @@ default LeaderRetrievalService getClusterRestEndpointLeaderRetriever() {
     void close() throws Exception;
```

Comment:
Keep the method to keep the JavaDoc. But we could make {{HighAvailabilityServices}} extend {{AutoCloseable}}.